### PR TITLE
Add amp-story doubleclick example

### PR DESCRIPTION
**Pull request by calebcordry**
I renamed the previous example to `Publisher Served` and introduced the new one as `DoubleClick`. They are very similar, with only the config as differences.

### CODE REVIEW

**Comment by calebcordry** [comment]
PTAL @sebastianbenz

**Comment by calebcordry** [comment]
cc/ @jasti

File: src/stories/30_Monetization/DoubleClick.html [hunk 1]
```diff
@@ -0,0 +1,353 @@
+<!--- {
+  "preview": "default"
+}--->
+
+<!--
+  ## Introduction
+  In an AMP story, you cannot put an `amp-ad` directly onto the page, instead, all
+  ads are fetched and displayed by the amp-story-auto-ads extension.
```
Comment (sebastianbenz) [inline src/stories/30_Monetization/DoubleClick.html]: nit: add code formatting for amp-story-auto-ads

File: src/stories/30_Monetization/DoubleClick.html [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+  on Github.
+-->
+<!doctype html>
+<html ⚡ lang="en">
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="https://ampbyexample.com/stories/monetization/amp-story-auto-ads/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <title>amp-story-auto-ads</title>
```
Comment (sebastianbenz) [inline src/stories/30_Monetization/DoubleClick.html]: Use the correct title

File: src/stories/30_Monetization/DoubleClick.html [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+
+  For the complete documentation please see our
+  [documentation](https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/amp-story-ads.md)
+  on Github.
+-->
+<!doctype html>
+<html ⚡ lang="en">
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="https://ampbyexample.com/stories/monetization/amp-story-auto-ads/">
```
Comment (sebastianbenz) [inline src/stories/30_Monetization/DoubleClick.html]: Please use the correct canonical URL

File: src/stories/30_Monetization/DoubleClick.html (hunk 1 above)
Comment (calebcordry) [reply src/stories/30_Monetization/DoubleClick.html]: done.

File: src/stories/30_Monetization/DoubleClick.html (hunk 2 above)
Comment (calebcordry) [reply src/stories/30_Monetization/DoubleClick.html]: done.

File: src/stories/30_Monetization/DoubleClick.html (hunk 3 above)
Comment (calebcordry) [reply src/stories/30_Monetization/DoubleClick.html]: done.

File: src/stories/30_Monetization/DoubleClick.html [hunk 4]
```diff
@@ -15,10 +15,10 @@
 <html ⚡ lang="en">
 <head>
   <meta charset="utf-8">
-  <link rel="canonical" href="https://ampbyexample.com/stories/monetization/amp-story-auto-ads/">
+  <link rel="canonical" href="https://ampbyexample.com/stories/monetization/DoubleClick/">
```
Comment (sebastianbenz) [inline src/stories/30_Monetization/DoubleClick.html]: s/DoubleClick/doubleclick

File: src/stories/30_Monetization/Publisher_Served.html [hunk 5]
```diff
@@ -15,10 +15,10 @@
 <html ⚡ lang="en">
 <head>
   <meta charset="utf-8">
-  <link rel="canonical" href="https://ampbyexample.com/stories/monetization/amp-story-auto-ads/">
+  <link rel="canonical" href="https://ampbyexample.com/stories/monetization/Publisher_Served/">
```
Comment (sebastianbenz) [inline src/stories/30_Monetization/Publisher_Served.html]: s/Publisher_Served/publisher_served

File: src/stories/30_Monetization/DoubleClick.html [hunk 6]
```diff
@@ -15,10 +15,10 @@
...
   <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
   <script async src="https://cdn.ampproject.org/v0.js"></script>
-  <title>amp-story-auto-ads</title>
+  <title>DoubleClick</title>
```
Comment (sebastianbenz) [inline src/stories/30_Monetization/DoubleClick.html]: Suggestion: use a title that better describes the content, e.g.: DoubleClick Ads for AMP Stories

File: src/stories/30_Monetization/Publisher_Served.html [hunk 7]
```diff
@@ -15,10 +15,10 @@
...
   <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
   <script async src="https://cdn.ampproject.org/v0.js"></script>
-  <title>amp-story-auto-ads</title>
+  <title>Publisher Served</title>
```
Comment (sebastianbenz) [inline src/stories/30_Monetization/Publisher_Served.html]: Same as above,  "Publisher served Ads for AMP Stories"

File: src/stories/30_Monetization/DoubleClick.html (hunk 6 above)
Comment (calebcordry) [reply src/stories/30_Monetization/DoubleClick.html]: done.

File: src/stories/30_Monetization/Publisher_Served.html (hunk 7 above)
Comment (calebcordry) [reply src/stories/30_Monetization/Publisher_Served.html]: done.

File: src/stories/30_Monetization/Publisher_Served.html (hunk 5 above)
Comment (calebcordry) [reply src/stories/30_Monetization/Publisher_Served.html]: done.

File: src/stories/30_Monetization/DoubleClick.html (hunk 4 above)
Comment (calebcordry) [reply src/stories/30_Monetization/DoubleClick.html]: done.

File: src/stories/30_Monetization/Publisher_Served.html [hunk 8]
```diff
@@ -172,7 +172,7 @@
       <!-- See the full example below:
        <div class="abe-story-preview">
           <amp-iframe media="(min-width: 650px)"
-              src="https://amp-by-example-api.appspot.com/stories/monetization/amp-story-auto-ads/index.html"
+              src="https://amp-by-example-api.appspot.com/stories/monetization/Publisher_Served/index.html"
```
Comment (sebastianbenz) [inline src/stories/30_Monetization/Publisher_Served.html]: s/Publisher_Served/publisher_served

Comment (calebcordry) [reply src/stories/30_Monetization/Publisher_Served.html]: done.

**Pull request merged**